ui/src/app/cohort-review/set-annotation-item/set-annotation-item.component.spec.ts

```diff
@@ -0,0 +1,146 @@
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {ReactiveFormsModule} from '@angular/forms';
+import {ActivatedRoute} from '@angular/router';
+import {ClarityModule} from '@clr/angular';
+import {Observable} from 'rxjs/Observable';
+
+import {ReviewStateService} from '../review-state.service';
+import {SetAnnotationItemComponent} from './set-annotation-item.component';
+
+import {
+  queryByCss,
+  updateAndTick,
+} from 'testing/test-helpers';
+
+import {
+  AnnotationType,
+  CohortAnnotationDefinition,
+  CohortAnnotationDefinitionService,
+} from 'generated';
+
+class StubRoute {
+  snapshot = {params: {
+    ns: 'workspaceNamespace',
+    wsid: 'workspaceId',
+    cid: 1
+  }};
+}
+
+class ApiSpy {
+  updateCohortAnnotationDefinition = jasmine.createSpy('updateCohortAnnotationDefinition');
+  deleteCohortAnnotationDefinition = jasmine.createSpy('deleteCohortAnnotationDefinition');
+  getCohortAnnotationDefinitions = jasmine.createSpy('getCohortAnnotationDefinitions');
+}
+
+describe('SetAnnotationItemComponent', () => {
+  let fixture: ComponentFixture<SetAnnotationItemComponent>;
+  let component: SetAnnotationItemComponent;
+
+  beforeEach(fakeAsync(() => {
+    TestBed
+      .configureTestingModule({
+        declarations: [
+          SetAnnotationItemComponent,
+        ],
+        imports: [
+          ClarityModule,
+          ReactiveFormsModule,
+        ],
+        providers: [
+          ReviewStateService,
+          {provide: CohortAnnotationDefinitionService, useValue: new ApiSpy()},
+          {provide: ActivatedRoute, useClass: StubRoute},
+        ],
+      }).compileComponents().then((resp) => {
+        fixture = TestBed.createComponent(SetAnnotationItemComponent);
+
+        // tick();
+        component = fixture.componentInstance;
+
+        // Default Inputs for tests
+        component.definition = <CohortAnnotationDefinition>{
```

Comment:
nit: is the cast needed here? I would have thought not (but not opposed if it's just for being explicit, mostly curious)